foundation/foundation-mda/src/main/resources/templates/data-delivery-data-records/dictionary.type.base.py.vm

```
@@ -45,9 +45,6 @@ class ${dictionaryType.capitalizedName}Base(ABC):
 #if ($dictionaryType.ethicsPolicy)
     ETHICS_POLICY: str = '${dictionaryType.ethicsPolicy}'
 #end
-#if ($dictionaryType.protectionPolicy)
```

Q: Would it be better to throw an exception here or a warning, or is that handled when the MDA json file is validated?  Like "protection policy is specified, but not supported.  Please remove protection policy from your definition"

---

The ideal is to clean up all the data encryption related code including the protection policy but yeah, I will definitely add a note in the release draft note to mention about the removal.

---

Also to clarify, because this property has been removed from the Schema, the model would fail to be read in entirely with a Fermenter exception. So the user would be notified that this property was removed well before even a single template was loaded/generated.